src/main/java/com/vesta/service/impl/UserServiceImpl.java

```diff
@@ -63,18 +74,31 @@ public void delete(Long id) {
         userRepository.deleteById(id);
     }
 
+    @Override
     public UserDto getByUsername(String username) {
-        return userConverter.convert(userRepository.findByUsername(username).orElse(null));
+        return userConverter.convert(userRepository.findByUsername(username)
+                .orElseThrow(() -> new UsernameNotFoundException("The username not found")));
     }
 
-    public Token login(AccountCredential accountCredential) {
-        if (userRepository.existsByUsernameOrEmailAndPassword(
-                accountCredential.getUsername(),
-                accountCredential.getEmail(),
-                accountCredential.getPassword()))
-            return tokenService.generatedToken(accountCredential.getUsername());
+    @Override
+    public Map<String, Token> login(AccountCredential accountCredential) {
+
+        UserEntity userEntity = userRepository
```

Comment:
I think this is common with line 79

Reply:
what do you mean?

Reply:
It was a conflict and i resolved it. I mean

Reply:
This code is the same for the line 79, so which mean you have duplicate code what isn't ok. Extract this code to a private method.
userRepository.findByUsername(accountCredential.getUsername())
                .orElseThrow(() -> new NotFoundException("The username or email doesn't exist"))